### ISSUE TITLE
[QUERY] Code samples for Azure.Messaging.EventHubs.Processor

### ISSUE THREAD

**Issue by alexanderbikk** [Azure/azure-sdk-for-net#18877]
**Query/Question**
I'm wondering why you remove the sample project for [Azure.Messaging.EventHubs.Processor](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/eventhub/Azure.Messaging.EventHubs.Processor/samples). I remember that you had good samples for this new library. But right now I can see only sketches, not real code samples. I see refactoring for the samples in commit history and the sample project was removed. 

I understand that new samples give a high-level overview. But as for me the previous sample where I can download and run the code was really useful and more informative.

From my point of view, it will be greater to see both high-level samples and sample projects with code for some real-world scenarios. For example process events in batch with checkpointing.

**Environment:**
 - Azure.Messaging.EventHubs.Processor

**Comment by jsquire** [comment]
Hi @alexanderbikk.  Thank you for your feedback and we regret that you're experiencing difficulties.   The format of samples was changed across the Azure SDK for .NET to focus more on discussion and highlighting of important snippets for key scenarios that an application is likely to encounter.  Feedback and user studies let us know that a large portion of our developers felt that the end-to-end samples required too much boilerplate code and made it difficult to locate the parts that were important to using the SDK in their own applications.

We're working on augmenting the markdown-based samples with stand-alone applications to provide the end-to-end illustration for key scenarios, but I don't have a timeline to share at this point.

In the mean time, you may want to reference the [Process events](https://github.com/Azure/azure-sdk-for-net/blob/65b19a062e246ceb283f89edefe4bc436d31f1e7/sdk/eventhub/Azure.Messaging.EventHubs.Processor/samples/Sample01_HelloWorld.md#process-events) portion of our introductory sample;  this is a reproduction of the core logic of the previous processor examples and can be pasted into a console application verbatim as a starting point.

**Comment by alexanderbikk** [comment]
HI @jsquire 
Thanks for your answer and your work. I agree that old samples had too much boilerplate code and required a deep dive especially when you are moving from the old version of SDK. 

But again for batch processing, you provided a much better example than you have right now. Good that I found it in commit history and already implemented ;)

My suggestion to have both high-level (easy to start) and deep-dive samples with more details. But I got the answer and seems like you are working on it. Greate!  Thanks! :)

Btw I would like to say that the new library is great since provides much more options than the previous one. But for me, it feels like the previous EvehtHubProcessor was quite simpler than the current version.
